### ISSUE TITLE
Temas armazenando path completo na configuração e não somente o nome do tema

### ISSUE THREAD

**Issue by jacsonp** [redelivre/delibera#74]
Os temas do delibera salvam o caminho inteiro até a pasta do tema na configuração e isso gera problemas nas migração e não é o correto a fazer, achei que já tinha resolvido isso a um tempo atrás, mas não está funcionando.

**Comment by jacsonp** [comment]
como vamos tirar os temas, da para invalidar aqui né @lpirola ?

**Comment by lpirola** [comment]
Sim, estou fazendo como conversamos, criei um tema em branco e depois removeremos os outros.
